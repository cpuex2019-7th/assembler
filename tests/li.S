li t0, 0x7fffffff
