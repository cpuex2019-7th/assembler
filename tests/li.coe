memory_initialization_radix=16;
memory_initialization_vector=80000537,fff50513;
radix=16;
