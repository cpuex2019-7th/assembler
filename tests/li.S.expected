lui t0, 0x80000000
addi t0, t0, 0xffffffff
